.github/workflows/pypi-publish.yml

```diff
@@ -37,11 +37,11 @@ jobs:
         mmv -v dist/'*.*.*-*' dist/'#1.#2.#3-'$timestamp'-#4'
```

Comment:
@christopher-mohr what was the rationale behind this timestamp? afaik that prevents bioconda from automatically bumping the version when a new release is published on pypi

Reply:
So you can create receips with conda skeleton :https://bioconda.github.io/contributor/guidelines.html#python

And all packages that I created via this are autobumped if the version of pypi is also bumped

Reply:
I don't see the connection to the time stamp at the moment. But in general I do not insist on keeping them.